src/main/resources/assets/create/lang/es_es.json

```diff
@@ -1527,10 +1527,10 @@
 	"block.create.mechanical_crafter.tooltip.control1": "Cuando se arranca por delante",
 	"block.create.mechanical_crafter.tooltip.action1": "_Circula la dirección_ hacia la que un mecanizador individual _mueve sus objetos_. Para formar una cuadrícula de trabajo, _organiza las correas en un flujo_ que mueva todos los objetos hacia un mecanizador final.  El mecanizador final debe _apuntar hacia fuera_ de la rejilla",
 	"block.create.mechanical_crafter.tooltip.control2": "Cuando se arranca hacia atrás",
-	"block.create.mechanical_crafter.tooltip.action2": "Conecta_ el _inventario de entrada_ de los mecanizadores adyacentes. Usa esto para _combinar ranuras_ en el grid de crafteo y _guardar el la entrada de trabajo_",
+	"block.create.mechanical_crafter.tooltip.action2": "Conecta_ el _inventario de entrada_ de los mecanizadores adyacentes. Usa esto para _combinar ranuras_ en la cuadrícula de trabajo y _guardar el la entrada de trabajo_",
 
 	"block.create.furnace_engine.tooltip": "MOTOR DEL HORNO",
-	"block.create.furnace_engine.tooltip.summary": "Una poderosa fuente de _Poder Rotatorio_ que requiere un _horno en funcionamiento_ para funcionar",
+	"block.create.furnace_engine.tooltip.summary": "Una poderosa fuente de _fuerza de rotación/torque_ que requiere un _horno en funcionamiento_ para funcionar",
```

Comment:
I have already merged your changes that were correct, it is not the same bronze and brass. :p. and I realized a serious mistake here, luckily! :)